Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py

```diff
@@ -84,13 +86,21 @@ def predict_phishing_words(model_name, model_store_type, email_subject, email_bo
     explain_result['NegativeWords'] = negative_words
     explain_result['OriginalText'] = tokenized_text_result['originalText'].strip()
     explain_result['TextTokensHighlighted'] = highlighted_text_markdown
+    predicted_label = explain_result["Label"]
 
     explain_result_hr = dict()
     explain_result_hr['TextTokensHighlighted'] = highlighted_text_markdown
-    explain_result_hr['Label'] = explain_result["Label"]
-    explain_result_hr['Probability'] = "%.2f" % explain_result["Probability"]
+    explain_result_hr['Label'] = predicted_label
+    explain_result_hr['Probability'] = "%.2f" % predicted_prob
     explain_result_hr['PositiveWords'] = ", ".join(positive_words)
     explain_result_hr['NegativeWords'] = ", ".join(negative_words)
+    if set_incidents_fields:
+        arg = {'customFields': {'DBotPrediction': predicted_label,
+                                'DBotPredictionProbability': predicted_prob,
+                                'DBotTextSuggestionHighlighted': highlighted_text_markdown}}
+        if original_incident_id is not None:
+            arg['id'] = original_incident_id
+        demisto.executeCommand("setIncident", arg)
```

Comment:
setIncident will perform on the current incident. just validate it's not playground